lib/iris/fileformats/netcdf.py

```diff
@@ -1,4 +1,4 @@
-# (C) British Crown Copyright 2010 - 2017, Met Office
+# (C) British Crown Copyright 2010 - 2018, Met Office
```

Comment:
These are fixed by https://github.com/SciTools/iris/pull/2932. Probably best to rebase when updating this PR.